executor-service/src/test/java/executor/service/service/step/impl/StepExecutionClickCssTest.java

```diff
@@ -0,0 +1,50 @@
+package executor.service.service.step.impl;
+
+import executor.service.model.Step;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+class StepExecutionClickCssTest {
+
+    private StepExecutionClickCss stepExecutionClickCss;
+    private WebDriver webDriver;
+
+    @BeforeEach
+    public void setUp() {
+        stepExecutionClickCss = new StepExecutionClickCss();
+
+    }
+
+    @AfterEach
+    public void tearDown() {
+        stepExecutionClickCss = null;
+        webDriver = null;
+    }
+
+    @Test
+    public void testGetStepAction() {
```

Comment:
Remove this 'public' modifier, please:
JUnit5 test classes and methods should have default package visibility

Reply:
Ohh. Thanks.